app/helpers/application_helper.rb

```diff
@@ -91,8 +91,6 @@ def visibility_icon(status)
       fa_icon('unlock', title: I18n.t('statuses.visibilities.unlisted'))
     elsif status.private_visibility? || status.limited_visibility?
       fa_icon('lock', title: I18n.t('statuses.visibilities.private'))
-    elsif status.direct_visibility?
-      fa_icon('envelope', title: I18n.t('statuses.visibilities.direct'))
     end
```

Comment:
why this change…?

Reply:
Ah, maybe a mistake. I thought we didn't render direct statuses in the public view. For some reason we don't have `statuses.visibilities.direct` in our strings.

Reply:
Ah yes, I had missed the translation was missing :/

The proper fix should probably to add that translation string and add back the icon, even if people do not commonly access DMs through this interface, it would be reassuring to them to see a clear indication that they are indeed DMs